packages/salesforce-adapter/src/filters/custom_objects_instances.ts

```diff
@@ -517,6 +532,14 @@ const filterTypesWithManyInstances = async (
   }
 }
 
+const getInaccessibleCustomFields = (objectType: ObjectType): string[] => (
+  Object.values(objectType.fields)
+    .filter(field => !isQueryableField(field))
+    .filter(field => !isSystemField(field)) // the value of system fields won't be visible anyway
```

Comment:
The terminology here is not very accurate. Some **System Fields** are both modifiable and are not hidden (e.g. **Name** and **CurrenyIsoCode**). I suggest we remove the wrapping method and inline it with

```suggestion
    // At the end of the fetch, the values of these fields won't be visible as we later hide read-only fields
    .filter(field => !isHiddenField(field) && !isReadOnlyField(field))
```

Reply:
Ack